### ISSUE TITLE
Add more default locations for docker-compose command

### ISSUE THREAD

**Issue by laurentpellegrino** [tobwiens/jsr223-docker-compose#12]
By default it is assumed that _docker-compose_ command is located in `/usr/local/bin/`. This location seems specific to some distributions. For instance, the default location on Fedora systems is `/usr/bin/`.

I know the location is configurable through property `docker.compose.command` but I think it would be nice to add support for default location on most standard systems like it is done for instance with Matlab and Scilab connectors if I remember correctly.

This could be done with the help of method that checks for the existance of the binary _docker-compose_ in a specified list of locations.

**Comment by tobwiens** [comment]
What if we leave the docker.compose.command field empty and the user needs to configure it. That will not favor any operating system.
And when the field is left empty, the docker-compose script engine is kind of "shut off" and does not write logs.

**Comment by laurentpellegrino** [comment]
In that case how the user will know that there is a missing configuration? he will just get an error?

Personnally, I prefer to make configuration as transparent and simple as possible for users. However, it's just my opinion. The best is to ask Brian.

**Comment by tobwiens** [comment]
Yes the user will get an error, something like docker-compose not found at "[PATH]".
If it is configured and there is no docker-compose installed it will give an error too.
